spec/indexers/descriptive_metadata/pub_year_1e_spec.rb

```diff
@@ -365,8 +365,8 @@
         }
       end
 
-      xit 'selects first date' do
-        expect(doc).to include('sw_pub_date_facet_ssi' => '')
+      it 'selects first date' do
+        expect(doc).to include('sw_pub_date_facet_ssi' => '2020')
```

Comment:
@arcadiafalcone is this the correct value?

Reply:
Yes.